flight/drone/drone_controller.py

```diff
@@ -34,6 +37,8 @@ class DroneController(object):
 
     Attributes
     ----------
+    task_factory : flight.tasks.TaskFactory
```

Comment:
I do not want the TaskFactory to be a part of the DroneController, it should only return a task object